toolkits/global/packages/global-autocomplete/demo/dist/index.html

```diff
@@ -62,10 +62,10 @@
 		}
 		input.removeEventListener('keyup', inputEvents);
 		if (container()) {
-			container().remove();
```

Comment:
I think our lint config specifies a preference for childNode.remove() over parentNode.removeChild(childNode), although it probably doesn't make any difference in this context? But am thinking if we runt the linter against the toolkits..

Reply:
@jon-stevens I ran `npm run lint` and it passed?

Reply:
@jon-stevens I actually tried updating to a more recent version of our linter config, but it gave me dozens or errors in countless files. I would suggest a separate PR for getting linting up to date?

Reply:
Yes, the same thing happened to me recently :(
Sure, makes sense.